cypress/integration/unit_tests/manage_disaster_set_score_parameters_test.js

```diff
@@ -39,8 +40,8 @@ describe('Score parameters-related tests for manage_disaster.js', () => {
         firstTest = false;
       }
     });
+    stateStub = cy.stub(ListEeAssets, 'getStateAssetsFromEe');
```

Comment:
use as notation for this stub? might as well start moving towards that?

Reply:
I'm not all in on `as` notation for stubs: I think it makes sense where the stub is being passed "around the back" between two functions, but nothing else needs to know about it. Also makes sense where the stub is being asserted on after test functions execute, so you want to enforce Cypress order. Neither is true for this stub, so prefer to leave it as is.

That being said, if you would like to have only one way that we do things, so generally get rid of these "let" instance variables and move everything to `as`, including for non-stubs, I'm open to it.

Reply:
I think it'd be nice to unify but it's low priority for me so fine to leave it this way